create Template Boilerplate

aceDavon opened the pull request:
Create the following boiletplate files
  - index.html,
  - .eslintrc.json
  - .hintrc
  - .stylelintrc.json
  - .README.md

aceDavon committed:
create Template Boilerplate

Create the following boiletplate files
  - index.html,
  - .eslintrc.json
  - .hintrc
  - .stylelintrc.json
  - .README.md

aceDavon committed:
Create banner and teams section on index.html

Create and style the banner and the teams sections on the index.html and
write corresponding styles

aceDavon committed:
Create Javascript functionalities

Add Javascript functionalities to allow for toggling of the navbar On
mobile version an feed the teams section dynamically from Javascript.

aceDavon committed:
Fix Linter errorrs accross all files

Kingstalux commented on index.html:
```diff
@@ -1,20 +1,320 @@
-    <h1>Hello, world!</h1>
+  <body class="col-12">
+    <header>
+      <nav
+        class="navbar navbar-expand-lg navbar-light bg-light py-2 px-sm-4 shadow"
+      >
+        <div class="container-fluid">
```
- [ ] Good job so far 👍 Kindly add the dark navbar as shown in the required design 
- Navbar:
![Capture](https://user-images.githubusercontent.com/85113567/181268478-4680c6de-b469-4a34-a4b4-6f26430340a8.PNG)

aceDavon replied:
Fixed

Kingstalux commented on index.html:
```diff
@@ -1,20 +1,320 @@
+          class="display-3 text-capitalize text-decoration-none fs-1 fw-bold"
+          >join us today!</a
+        >
+        <p class="display-5 fs-5">
+          Together, let's secure a future for our generations yet to come
+        </p>
+      </section>
```
- [ ] It would be great if you could add some more padding or margin to this section so as to match the original design.
- Your design:
![Capture1](https://user-images.githubusercontent.com/85113567/181268746-f720d862-547e-4f81-92d3-aeae20c5ab52.PNG)
- Required design:
![Capture2](https://user-images.githubusercontent.com/85113567/181268791-0f406b61-0116-48a2-8e74-2d4a182db139.PNG)

aceDavon replied:
resolved

Kingstalux commented on index.html:
```diff
@@ -1,20 +1,320 @@
+        <p class="display-5 fs-5">
+          Together, let's secure a future for our generations yet to come
+        </p>
+      </section>
+      <section class="bg-dark d-flex flex-column gap-4 services px-sm-2 py-3">
+        <h3 class="h3 text-center">Our Services</h3>
+        <div class="line"></div>
+        <ul
+          class="d-md-flex d-block gap-1 flex-grow-1 flex-wrap justify-content-center"
+        >
+          <li class="d-flex flex-row flex-sm-column gap-2 py-2">
+            <img
+              src="./img/cattle.jpeg"
+              alt="cattle-images"
+              class="rounded-circle"
+            />
+            <p class="h5 fw-bold text-center inner-heading">Livestock Farms</p>
```
- [ ] Kindly add the padding in this section as well as the hover effect to all your cards. This will improve the UI of your site 💯 
- [ ] Also kindly use the correct colors for all your heading. For example in this section, the heading should be white but yours is red. Kindly rectify this issue throughout your site 👍 
- Your design:
![Capture3](https://user-images.githubusercontent.com/85113567/181269089-d587f908-ecee-484f-a2ec-ba06f5fccf8f.PNG)
- Required design:
![Capture4](https://user-images.githubusercontent.com/85113567/181269138-5904416e-b700-4fed-9f8a-95956161084c.PNG)

aceDavon replied:
Resolved

Kingstalux commented on index.html:
```diff
@@ -1,20 +1,320 @@
+          <img src="./img/insurance.png" alt="insuranceIcon" />
+          <img src="./img/google.webp" alt="googleIcon" />
+        </div>
+      </section>
```
- [ ] It would also be good to add more padding to this section too as well as use the correct color for its heading 😄 
- Your design:
![Capture5](https://user-images.githubusercontent.com/85113567/181269860-6f5d629a-3040-4641-8dc3-f57120e55465.PNG)
- Required design:
![Capture7](https://user-images.githubusercontent.com/85113567/181269914-9cdf5a65-39ff-4b96-9b8f-7d99cd934e35.PNG)

aceDavon replied:
Resolved

Kingstalux commented on index.html:
```diff
@@ -1,20 +1,320 @@
+        </div>
+      </section>
+    </main>
+    <footer class="bg-light d-none d-lg-flex gap-4 col-12 py-3 px-5 d-none">
+      <div class="col-5 py-3">
+        <img src="./img/logo.png" class="w-25" alt="logo" />
+        Hevons Synergy
+      </div>
+      <div class="col-6 px-2 py-3">
+        &copy; 2022, Some rights reserved
+        <p class="fst-itealic fs-small fw-light">
+          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nisi
+          voluptatem sapiente, dolor quia error numquam blanditiis voluptatibus
+          quam? Qui optio vitae ad molestias expedita consectetur!
+        </p>
```
- [ ] For the footer, kindly align your items to the center. Right now it looks as if the text is above the logo. It would be great if you could address this issue.
- Your design:
![Capture6](https://user-images.githubusercontent.com/85113567/181270243-9163575e-751d-43d9-a8e3-ac8b4f47a263.PNG)

Kingstalux commented on README.md:
```diff
@@ -7,9 +7,9 @@
+- Bootstrap
+- Git, github
 
 ## Live Demo (if available)
```
- [ ] Kindly deploy your project and provide a working live link in your README file. Your current live demo link is not working.

aceDavon replied:
Resolved

Kingstalux commented on pages/about.html:
```diff
@@ -100,28 +100,79 @@
           </p>
         </div>
       </section>
+      <section class="bg-white py-4">
+        <h1 class="h1 text-uppercase text-center mt-5 fw-bold">Our Values</h1>
+        <div class="about-line"></div>
+        <p class="text-center text-dark mt-3 col-sm-10 col-md-8 mx-auto">
+          We are a company built on solid cultural orientatons and believes. Our
+          goal is to ensure the availability, affordability of both raw and
+          processed/market ready food items.
+        </p>
+        <div class="shadow-lg col-6 mx-auto">
```
- [ ] Kindly arrange the alignment of your items on your about page for the desktop version 👍 
- Your site:
![Capture](https://user-images.githubusercontent.com/85113567/181271710-a87692f6-ae80-460f-9909-0c0c8f7a5c87.PNG)
- Required design:
![Capture1](https://user-images.githubusercontent.com/85113567/181271761-1a8da2f8-352e-4e4f-bd74-0040e7eba053.PNG)

aceDavon replied:
Fixed

Kingstalux commented on pages/about.html:
```diff
@@ -100,28 +100,79 @@
+        <img src="../img/google.webp" alt="googleIcon" />
+      </div>
+    </section>
+    <footer class="bg-light bg-lg-dark text-dark d-flex gap-4 col-12 py-3 px-5">
+      <div class="col-5 py-3">
+        <img src="../img/logo.png" class="w-25" alt="logo" />
+        Hevons Synergy
+      </div>
+      <div class="col-6 px-2 py-3">&copy; 2022, Some rights reserved</div>
+    </footer>
```
- [ ] In the desktop version of your about page, the footer should be dark. Kindly correct this issue 💯 
- Your design:
![Capture2](https://user-images.githubusercontent.com/85113567/181272000-810370e8-1f4e-45c8-8fef-f1610b04115b.PNG)
- Required design:
![Capture3](https://user-images.githubusercontent.com/85113567/181272037-db469c6b-a920-49c4-ab36-675c7080b092.PNG)

aceDavon replied:
Fixed

Kingstalux commented on pages/about.html:
```diff
@@ -100,28 +100,79 @@
         </ol>
       </nav>
```
- [ ] A user should be able to navigate between the home and about pages but that is not the case now. Kindly rectify this issue so that when a user clicks the about link on the home page, the user is taken to the about page and vice versa. This should also be applied to the mobile menu 💯 
- [ ] Also, the about page should have a dark navbar. Kindly add this too 👍

aceDavon replied:
Fixed

Kingstalux reviewed (changes requested):
Hi @aceDavon, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 👏 
✅ Good JavaScript functionality 👍 
✅ Linters are passing 👌 
✅ Good commit history 👏 



## Required Changes ♻️
_Check the comments under the review._
- [ ] Kindly record a video where you present your site and all the key features of your site as well as a piece of interesting code you worked on while building this site. The video should have a maximum length of 5 minutes 👍 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

aceDavon committed:
Resolve Issues raised from review

Acknowledge and fix issues as suggeted by the reviewer on both the home,
and the about pages

aceDavon committed:
Fix paddings on navbar

aceDavon committed:
Fix stylelint errors on style.css

aceDavon committed:
Fix a typo on README.md file

aceDavon committed:
Fix broken live link on README.md

BertrandConxy reviewed (approved):
Hello @aceDavon ✋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🥳 🍾 💐 

## Highlights 💪 
- innovative idea in a good-looking UI design
- no linter errors
- good PR description
- Good improvements based on previous reviewers' suggestions

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
